plugins/kiali-common/src/helpers/config.ts

```diff
@@ -0,0 +1,51 @@
+import { Config } from '@backstage/config';
+
+import { KialiDetails } from '../types';
+
+const KIALI_PREFIX = 'catalog.providers.kiali';
```

Comment:
Please provide a config schema. You can see how we do it in other backend plugins:

https://github.com/janus-idp/backstage-plugins/blob/main/plugins/ocm-backend/config.d.ts

https://backstage.io/docs/conf/defining/